bindings/java/src/main/java/org/apache/opendal/OperatorOutputStream.java

```diff
@@ -35,31 +35,38 @@ protected void disposeInternal(long handle) {
         }
     }
 
-    private static final int MAX_BYTES = 16384;
+    private static final int DEFAULT_MAX_BYTES = 16384;
 
     private final Writer writer;
-    private final byte[] bytes = new byte[MAX_BYTES];
+    private final byte[] bytes;
+    private final int maxBytes;
 
     private int offset = 0;
 
     public OperatorOutputStream(Operator operator, String path) {
+        this(operator, path, DEFAULT_MAX_BYTES);
+    }
+
+    public OperatorOutputStream(Operator operator, String path, int maxBytes) {
         final long op = operator.nativeHandle;
         this.writer = new Writer(constructWriter(op, path));
+        this.maxBytes = maxBytes;
+        this.bytes = new byte[maxBytes];
```

Comment:
There are two aspects here:

- One is the Java-side byte buffer, which is used to reduce JNI overhead.
- The other is the Rust core-side `chunk`, which buffers user input.

Is it a good idea to set the writer's chunk at the same time?

Reply:
> The other is the Rust core-side chunk, which buffers user input.

Could you point out the related code or examples? We may do this in a follow-up PR.

Reply:
https://docs.rs/opendal/latest/opendal/operator_functions/struct.FunctionWriter.html#method.chunk